pkg/crd/markers/validation.go

```diff
@@ -635,3 +659,44 @@ func (m XValidation) ApplyToSchema(schema *apiext.JSONSchemaProps) error {
 	})
 	return nil
 }
+
+func (fields AtMostOneOf) ApplyToSchema(schema *apiext.JSONSchemaProps) error {
+	if len(fields) == 0 {
+		return nil
+	}
+	rule := fieldsToOneOfCelRuleStr(fields)
+	xvalidation := XValidation{
+		Rule:    fmt.Sprintf("%s <= 1", rule),
+		Message: fmt.Sprintf("at most one of the fields in %v can be set", fields),
```

Comment:
Just to be consistent with the wording API reviewers typically use, can we change this to `at most one of the fields in %v may be set`. If you check the API conventions docs they use `may` and `must` often over words like `can`